src/main/java/bhs/devilbotz/commands/assist/AutoScore.java

```diff
@@ -38,10 +38,12 @@ public AutoScore(Arm arm, DriveTrain drivetrain, Gripper gripper) {
     super();
     addCommands(CommandDebug.start());
     addCommands(new DriveStraightPID(drivetrain, -DriveConstants.POSITION_DRIVE_FROM_PORTAL));
+    addCommands(drivetrain.stop());
     addCommands(new ArmToPosition(arm, ArmConstants.POSITION_TOP, gripper));
     addCommands(new DriveStraightPID(drivetrain, DriveConstants.POSITION_DRIVE_FROM_PORTAL));
+    addCommands(drivetrain.stop());
     addCommands(new ArmMoveDistance(arm, ArmConstants.POSITION_SCORING_DELTA, gripper));
     addCommands(new GripperOpen(gripper));
-    addCommands(CommandDebug.start());
+    addCommands(CommandDebug.end());
```

Comment:
I see this typo in a few other commands.  Do you want to add those here too? Or fix in sep PR to keep this one smaller?

Reply:
Thanks. I'll look for this error and fix in all the other places, too.

Reply:
I couldn't find this error anywhere else.

Reply:
sorry, must of been looking at wrong branch/code or out of date code